[2024-01-15T13:08:51.647-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_novo.df scheduled__2024-01-15T16:07:23.929758+00:00 [queued]>
[2024-01-15T13:08:51.650-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_novo.df scheduled__2024-01-15T16:07:23.929758+00:00 [queued]>
[2024-01-15T13:08:51.650-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-15T13:08:51.660-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): df> on 2024-01-15 16:07:23.929758+00:00
[2024-01-15T13:08:51.664-0300] {standard_task_runner.py:57} INFO - Started process 42892 to run task
[2024-01-15T13:08:51.666-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_novo', 'df', 'scheduled__2024-01-15T16:07:23.929758+00:00', '--job-id', '633', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpagyv9wge']
[2024-01-15T13:08:51.666-0300] {standard_task_runner.py:85} INFO - Job 633: Subtask df
[2024-01-15T13:08:51.682-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_novo.df scheduled__2024-01-15T16:07:23.929758+00:00 [running]> on host desktop
[2024-01-15T13:08:51.708-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_novo' AIRFLOW_CTX_TASK_ID='df' AIRFLOW_CTX_EXECUTION_DATE='2024-01-15T16:07:23.929758+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-15T16:07:23.929758+00:00'
[2024-01-15T13:08:53.917-0300] {logging_mixin.py:154} INFO - Called | 1705334933.9148464 | df
[2024-01-15T13:08:53.924-0300] {logging_mixin.py:154} INFO - SAVING FILE df
[2024-01-15T13:08:53.925-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-15T13:08:53.927-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_novo, task_id=df, execution_date=20240115T160723, start_date=20240115T160851, end_date=20240115T160853
[2024-01-15T13:08:54.000-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-15T13:08:54.009-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
